<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #ffffff;
}

QProgressBar {
	border-color: 0px solid red;
    background: #ebedef;
    border-top-right-radius: 32px;
	border-top-left-radius: 32px;
	border-bottom-right-radius: 32px;
	border-bottom-left-radius: 32px;
}

QProgressBar::chunk {
	background: #cc5c5c;
    border-top-right-radius: 32px;
	border-top-left-radius: 32px;
	border-bottom-right-radius: 32px;
	border-bottom-left-radius: 32px;
}</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 0;
   color: #ffffff;
   height: 30px;
   margin-top: 30px;
}</string>
   </property>
   <widget class="QWidget" name="container" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>670</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>671</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="header_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="gsreg_logotype">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="julia_logotype">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="line_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="Line" name="header_line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="slider_layout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QProgressBar" name="progress">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="title_layout">
        <item>
         <widget class="QLabel" name="page_step">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="page_title">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Step 0: Lalala</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="step_layout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="nav_layout">
        <item>
         <layout class="QHBoxLayout" name="nav_buttons_layout">
          <item>
           <widget class="QPushButton" name="prev_button">
            <property name="styleSheet">
             <string notr="true">QPushButton {
   background: #6682e0;
}
QPushButton:hover {
   background: #3a5fcc;
}</string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="nav_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>400</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="next_button">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#next_button {
	background: #60ad51;
}
QPushButton#next_button:hover {
   background: #228a22;
}</string>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>gsreg_logotype</zorder>
   <zorder>container</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
